src/test/java/uk/gov/justice/hmpps/prison/service/OffenderNonAssociationsServiceTest.java

```diff
@@ -1,231 +0,0 @@
-package uk.gov.justice.hmpps.prison.service;
-
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.Mock;
-import org.mockito.junit.jupiter.MockitoExtension;
-import uk.gov.justice.hmpps.prison.api.model.OffenderNonAssociation;
-import uk.gov.justice.hmpps.prison.api.model.OffenderNonAssociationDetail;
-import uk.gov.justice.hmpps.prison.api.model.OffenderNonAssociationDetails;
-import uk.gov.justice.hmpps.prison.repository.jpa.model.AgencyInternalLocation;
-import uk.gov.justice.hmpps.prison.repository.jpa.model.AgencyLocation;
-import uk.gov.justice.hmpps.prison.repository.jpa.model.NonAssociationReason;
-import uk.gov.justice.hmpps.prison.repository.jpa.model.NonAssociationType;
-import uk.gov.justice.hmpps.prison.repository.jpa.model.Offender;
-import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderBooking;
-import uk.gov.justice.hmpps.prison.repository.jpa.repository.OffenderBookingRepository;
-import uk.gov.justice.hmpps.prison.repository.jpa.repository.OffenderNonAssociationDetailRepository;
-
-import java.time.LocalDateTime;
-import java.util.List;
-import java.util.Optional;
-
-import static java.util.Collections.emptyList;
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
-import static org.mockito.Mockito.when;
-
-@ExtendWith(MockitoExtension.class)
-public class OffenderNonAssociationsServiceTest {
```

Comment:
deleting this file doesn't seem right - sounds like it should be repurposed to call `retrieveByOffenderNo` instead

Reply:
yes